src/EnergyPlus/PhotovoltaicThermalCollectors.hh

```diff
@@ -150,14 +201,15 @@ namespace PhotovoltaicThermalCollectors {
         bool MySetPointCheckFlag;
         bool MyOneTimeFlag;
         bool SetLoopIndexFlag;
+        Real64 QdotSource; // Source/sink term
 
         // Default Constructor
         PVTCollectorStruct()
-            : Type(DataPlant::PlantEquipmentType::Invalid), WPlantLoc{}, EnvrnInit(true), SizingInit(true), PVTModelType(0), SurfNum(0), PVnum(0),
-              PVfound(false), WorkingFluidType(WorkingFluidEnum::LIQUID), PlantInletNodeNum(0), PlantOutletNodeNum(0), HVACInletNodeNum(0),
-              HVACOutletNodeNum(0), DesignVolFlowRate(0.0), DesignVolFlowRateWasAutoSized(false), MaxMassFlowRate(0.0), MassFlowRate(0.0),
-              AreaCol(0.0), BypassDamperOff(true), CoolingUseful(false), HeatingUseful(false), MySetPointCheckFlag(true), MyOneTimeFlag(true),
-              SetLoopIndexFlag(true)
+            : Type(DataPlant::PlantEquipmentType::Invalid), WPlantLoc{}, EnvrnInit(true), SizingInit(true), SurfNum(0), PVnum(0), PVfound(false),
```

Comment:
Initialize in declaration.

Reply:
Most of these variables were already there? I can help with existing code changes if needed.

Reply:
Yeah, most of them were already there.  We are trying to move away from this type of initializer list constructor because it makes for weird diffs and is more difficult to see how a variable is initialized.  You don't have to go change everything already there, but if you are adding new members to a struct, we prefer that you at least initialize those new variables right where they are declared on the class.  This isn't an absolute show stopper but if you have any more changes to make, it would be nice to add this to them.